src/main/java/uk/gov/hmcts/ccd/domain/types/BaseTypeValidator.java

```diff
@@ -19,7 +19,14 @@
     default Boolean isNullOrEmpty(final JsonNode dataValue) {
         return dataValue == null
             || dataValue.isNull()
-            || (dataValue.isTextual() && (null == dataValue.asText() || dataValue.asText().trim().length() == 0))
+            || (dataValue.isTextual() && (null == dataValue.asText() || dataValue.asText().length() == 0))
+            || (dataValue.isObject() && dataValue.toString().equals("{}"));
+    }
+
+    default Boolean isNull(final JsonNode dataValue) {
+        return dataValue == null
+            || dataValue.isNull()
+            || (dataValue.isTextual() && (null == dataValue.asText()) )
             || (dataValue.isObject() && dataValue.toString().equals("{}"));
```

Comment:
are we sure it is right to interpret empty data value object as null? I feel the last line does not belong in this function